### PR TITLE
[github] added modified Radium PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by hiergaut**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Check format style with the script provided in `scripts/source-format-all.sh`
+- You must use clang-format style (github CI version is 11)
+_These checks are enforced by github workflow actions_
+_Please refer to the corresponding log in case of failure_
+
+## UPDATE the form below to describe your PR
+
+Please check if the PR fulfills these requirements
+
+- [ ] The commit message follows our guidelines
```
Comment (nmellado) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Do we have guidelines for the commits ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Pull Request Desription
+
+## Check if you branch history is PR compatible
+
+- Your branch need to be up to date with origin/main
+- Update your git repository `git fetch origin` if origin is this remote
+- Check format style with the script provided in `scripts/source-format-all.sh`
+- You must use clang-format style (github CI version is 11)
```
Comment (nmellado) [inline .github/PULL_REQUEST_TEMPLATE.md:8]:
```suggestion
- You must use clang-format style (github CI version is 11)
- Your proposal must include a commit where the file CHANGELOG is updated (add a line describing the proposed change + PR id, see file for examples). If your contribution is significant, you can also ask to be added to the contributor list in the discussion.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (hiergaut) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: Yes, the guidelines for commit is in CONTRIBUTING.md file. (line 141)

Comment (nmellado) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Please link to this file in the PR template

**Comment by nmellado** [comment]
Donc forget to update the CHANGELOG

**Comment by hiergaut** [comment]
I have committed your proposal.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (hiergaut) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Can we refer to other md file in md file ?

Comment (hiergaut) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
- [ ] The commit message follows our guidelines (see CONTRIBUTING)
```

**Review by nmellado — CHANGES_REQUESTED** [review]
Please rebase on `main` + squash the commits affecting the template (but leave the CHANGELOG in a separated commit)

**Comment by hiergaut** [comment]
Done for me if tests pass

**Review by nmellado — APPROVED** [review]

**Pull request merged**